install/build_from_source.md

```diff
@@ -56,7 +56,42 @@ To have ROOT setup automatically at each login, that command can be appended to
 
 ## ROOT, Python and PyROOT
 
-> TODO: explain what build options are relevant, what they do, Python2/Python3, etc.
+ROOT is built with Python bindings (PyROOT) if the option `-Dpyroot=ON` is present (enabled by default). If the old version of PyROOT (pre-v6.22) wants to be built, the option `-Dpyroot_legacy` needs to be specified.
```

Comment:
this is a bit confusing, users will have no idea what the "old version of PyROOT" is. I would suggest to remove this from here...maybe link to a separate page "ROOT, Python and PyROOT in versions <6.22" or something like that? or whatever place explains the transition

Reply:
made two separate sections